src/Pact/Analyze/Eval/Pure.hs

```diff
@@ -0,0 +1,256 @@
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE LambdaCase          #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+module Pact.Analyze.Eval.Pure where
```

Comment:
We should rename this `Core` to match the type. Also the name of this file

Reply:
fixed